src/com/luxoft/java/course/shevtsov/homework/algorithm/additional/NumberSymmetryCheck.java

```diff
@@ -0,0 +1,31 @@
+package com.luxoft.java.course.shevtsov.homework.algorithm.additional;
+
+public class NumberSymmetryCheck {
+    private final static int NUMBER_HALF_LENGTH = 2;
+    public final static int NUMBER_LENGTH = NUMBER_HALF_LENGTH * 2;
+    private static int number1 = 0;
+
+    public static boolean checkNumberSymmetry(int number){
```

Comment:
It is not actually number symmetry, because you compare sums of numbers

Reply:
agree. corrected